cmd/server.go

```diff
@@ -79,6 +82,23 @@ func (s *Server) getRole(IP string) (string, error) {
 	return role, nil
 }
 
+func (s *Server) debugStoreHandler(w http.ResponseWriter, r *http.Request) {
+	output := make(map[string]interface{})
+
+	output["rolesByIP"] = s.store.DumpRolesByIP()
+	output["rolesByNamespace"] = s.store.DumpRolesByNamespace()
+	output["namespaceByIP"] = s.store.DumpNamespaceByIP()
+
+	o, err := json.Marshal(output)
+	if err != nil {
+		log.Errorf("Error converting debug map to json: %+v", err)
+	}
+
+	if _, err := w.Write(o); err != nil {
+		log.Errorf("Error writing response: %+v", err)
```

Comment:
This is unnecessary - most likely client has just disconnected and there is no point logging that. Not on `error` in any case.